Gemfile

```diff
@@ -13,7 +13,7 @@ group :development, :test do
   gem "byebug"
   gem "ci_reporter_rspec"
   gem "climate_control"
-  gem "database_cleaner"
+  gem "database_cleaner-mongoid"
```

Comment:
Seems sensible, but I think the commit message needs tweaking (ie release 2.0.0 of what?)